### PR TITLE
Update protobufs for 0.13.0-alpha.2

### PR DESCRIPTION
**Pull request by tinker-michaelj**
Signed-off-by: tinker-michaelj <michael.tinker@hedera.com>

**Closed PR in [Hedera protobufs](https://github.com/hashgraph/hedera-protobufs)**
- [x] https://github.com/hashgraph/hedera-protobufs/pull/15

### CODE REVIEW

File: src/main/proto/ScheduleSign.proto [hunk 1]
```diff
@@ -28,14 +28,16 @@ option java_multiple_files = true;
...
+Upon <tt>SUCCESS</tt>, the receipt includes the <tt>scheduledTransactionID</tt> to use to query for the 
+record of the scheduled transaction's execution (if it occurs). 
+
+Other notable response codes include <tt>INVALID_SCHEDULE_ID</tt>, <tt>SCHEDULE_WAS_DELETD</tt>, 
+<tt>INVALID_ACCOUNT_ID</tt>, <tt>UNRESOLVABLE_REQUIRED_SIGNERS</tt>, <tt>SOME_SIGNATURES_WERE_INVALID</tt>, 
+and <tt>NO_NEW_VALID_SIGNATURES</tt>. For more information please see the section of this 
+documentation on the <tt>ResponseCode</tt> enum. */
 message ScheduleSignTransactionBody {
-  ScheduleID scheduleID = 1; // The ID of the Scheduled entity
-  SignatureMap sigMap = 2; // The signature map containing the signature(s) to authorise the transaction
```
Comment (ljianghedera) [inline src/main/proto/ScheduleSign.proto:40]: If the `sigMap`  is removed, how will the signatures be provided?

**Review by ljianghedera — APPROVED** [review]
LGTM except the small question.

File: src/main/proto/ScheduleSign.proto (hunk 1 above)
Comment (tinker-michaelj) [reply src/main/proto/ScheduleSign.proto:40]: Via the top-level `sigMap`...the [spec](https://github.com/hashgraph/hedera-services/blob/master/docs/scheduled-transactions/revised-spec.md) has more details 😁

Comment (ljianghedera) [reply src/main/proto/ScheduleSign.proto:40]: Yeah. I believe there will be some way to provide the signatures. I will take a loot. Thx.

**Pull request merged**